types/visualne/index.d.ts

```diff
@@ -0,0 +1,48 @@
+// Type definitions for visualne 1.0
+// Project: https://github.com/vamidi/storytime
```

Comment:
```suggestion
// Project: https://github.com/vamidi/visualne
```

Reply:
I think you are right. I think it only sounded cool to be part of the @types group. I also think that it is probably faster to ship instead of being with the @types repo?

Reply:
Easier for users, too, since they won't have to remember to `npm i @types/visualne`.